src/containers/AdminCampaignEdit.jsx

```diff
@@ -554,6 +566,7 @@ export class AdminCampaignEditBase extends React.Component {
         expandableBySuperVolunteers: false
       }
     ];
+    // console.log("final sections", finalSections);
```

Comment:
I really thought I would find a contact here.  Figuring out to put the console.log here to see how the final sections appear, it was exciting when I could spot them in the console.  However, the contact section doesn't save the contact data in a way that I found useful.